components/gateway/cmd/gateway/gateway.go

```diff
@@ -50,18 +46,9 @@ func main() {
 	}
 
 	internalHandler := newInternalHandler(serviceDefinitionService, nameResolver, proxyCache, options.skipVerify, options.proxyTimeout)
-	externalHandler := externalapi.NewHandler()
```

Comment:
Mind that externalHandler is still needed as it handles health endpoint used by readiness probe

Reply:
@akgalwas done.